CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+Fix central manage login and logout bugs
```

Comment:
Add it into a `Fixed` group.

You can obtain more info [here](http://keepachangelog.com/en/1.0.0/)